pkgs/by-name/ph/photonvision/package.nix

```diff
@@ -3,25 +3,25 @@
   stdenv,
   fetchurl,
   makeWrapper,
-  temurin-jre-bin-11,
+  temurin-jre-bin-17,
   bash,
   suitesparse,
   nixosTests,
 }:
 
 stdenv.mkDerivation rec {
   pname = "photonvision";
-  version = "2024.3.1";
+  version = "2025.2.1";
 
   src =
     {
       "x86_64-linux" = fetchurl {
         url = "https://github.com/PhotonVision/photonvision/releases/download/v${version}/photonvision-v${version}-linuxx64.jar";
-        hash = "sha256-t9drkGFA3IurZqWAkzEaONVJkp5JHMEFBBW50r+SD68=";
+        hash = "sha256-yEb6GCt29DjZNDsIqDvF/AiCw3QVMxUFKQM22OlMl7Q=";
       };
       "aarch64-linux" = fetchurl {
         url = "https://github.com/PhotonVision/photonvision/releases/download/v${version}/photonvision-v${version}-linuxarm64.jar";
-        hash = "sha256-ninCVxse0x6lBA2NL3kwMeuHAeNzSa9rdP2dnmMNFgc=";
+        hash = "sha256-mNQk8gaTASsmyJUpLLIbG7QRMjbdSN2LMCXx6j3gbCU=";
```

Comment:
@flashrun24 Do you know what happened here? The hash had to be updated to match in https://github.com/NixOS/nixpkgs/pull/461902